### DISCUSSION TITLE
Why can't I access basic data with the client credentials?

### DISCUSSION THREAD

**Discussion by uravt** [spotify-web-api-java/spotify-web-api-java#320] [Q&A]
I used the client credentials example code and combined it with one of the examples for getAlbum. It returns an invalid token error , which I don't understand as this is non user specific data. Am I doing something wrong?
Thanks for the help!

**Comment by dargmuesli** [comment]
> one of the examples

Please always be as specific as possible for people to be able to assist. Show your actual code for example.

> It returns an invalid token error

It's not a surprise but it appears the token is invalid. This repository is only meant to provide a wrapper for Java, not general API usage support. For that, please head over to the [Spotify community](https://community.spotify.com/)!
